astropy/io/ascii/tests/common.py

```diff
@@ -35,3 +35,8 @@ def assert_almost_equal(a, b, **kwargs):
 
 def assert_true(a):
     assert a
+
+
+def assert_equal_splitlines(arg1, arg2):
+    __tracebackhide__ = True
+    assert_equal(arg1.splitlines(), arg2.splitlines())
```

Comment:
```suggestion
    __tracebackhide__ = True
    assert arg1.splitlines() == arg2.splitlines()
```
As the above will cause problems once `assert_equal` is removed in #15839 might be best to inline that function already now, or alternatively use `np.testing.assert_equal` here, I guess (obviating the need for an extra `__tracebackhide__`). Again hard to say which one is better in terms of a more useful traceback; since `assert_equal_splitlines` is mostly used on small tables here with not all columns numerical, I'd tend to the simple `assert`.